core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -216,6 +217,8 @@ object TestUtils extends Logging {
 
     if (protocolAndPorts.exists { case (protocol, _) => usesSslTransportLayer(protocol) })
       props.putAll(sslConfigs(Mode.SERVER, false, trustStoreFile, s"server$nodeId"))
+    if (protocolAndPorts.exists { case (protocol, _) => usesSaslTransportLayer(protocol) })
+      props.put(KafkaConfig.SaslKerberosServiceNameProp, "kafka")
```

Comment:
@fpj serviceName is not a standard property of the Kerberos module and IBM Kerberos module throws an exception if this property is specified. The alternative configuration option for serviceName works with any JDK.